changelog/unreleased/insecure-options.md

```diff
@@ -0,0 +1,6 @@
+Enhancement: Make insecure options configurable
+
+We had several hard-coded 'insecure' flags. These options are now configurable but still default to true until the GA release.
```

Comment:
Why not changing them to false?

Reply:
I thought it would be annoying to have to set them to false every time. On the other hand. Every developer could set those env vars globally once and then forget about it. :thinking: 

Yeah, let me change them to false.

Reply:
We also have eg. `PROXY_OIDC_INSECURE` defaulting to true. Should we add an `OCIS_URL` like `OCIS_INSECURE` option? 
You're right, that it will be really annoying to set all these *_INSECURE options if you're on a dev environment or on an internal network

Reply:
So I changed it to `false` per default. Going with the secure solution on default is better.
IMO we shouldn't introduce something like `OCIS_INSECURE` as you're not seeing what is being affected by this flag and this could have some unwanted security implications.

Reply:
We then should at least add them to the .vscode/launch.json and to the deployment examples (I will check if they break)